gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/RawResourceController.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Coveo, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.gate.controllers;
+
+import com.netflix.spinnaker.gate.services.RawResourceService;
+import groovy.transform.CompileStatic;
+import io.swagger.annotations.ApiOperation;
+import java.util.List;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@CompileStatic
+@RestController
+public class RawResourceController {
+
+  @Autowired RawResourceService rawResourceService;
+
+  @ApiOperation(
+      value = "Retrieve a list of raw resources for a given application",
+      response = List.class)
+  @RequestMapping(value = "/applications/{application}/rawResources", method = RequestMethod.GET)
```

Comment:
nit, curious about `/rawResources` or `/resources`.

Reply:
As implemented here: https://github.com/spinnaker/clouddriver/commit/bd351149f4fab0a187ae55ece3d52476416ea7b4

Reply:
Sure, same comment would apply there.

The `raw` prefix just struck me as a little strange that's all. I don't understand what's "raw" about it, it seems more like all resources which would normally just be `/resources`.

I don't mind the idea of a general purpose resources endpoint that you could filter down to what you'd like.

Reply:
One way to see it is that they're raw in the sense that they're not bucketed/casted as server groups/firewalls/load balancers. It does seem weird without the whole context though. In the first iteration of my PR, the entire (raw) data that made up the resources was served by the clouddriver API. However, that data (the resources' manifest) is no longer sent in the payload due to potential performance issues/future caching changes. The resources are no longer "raw" in that sense.

I'm just an outsider here, naming is completely at your convenience and it's the right time to change this if another name would be better (it's not used yet). Should I change it?
@ezimanyi, thoughts?

Reply:
I don't have a strong opinion one way or another.  From my perspective the `raw` prefix made sense because this is returning all resources in the Kubernetes cluster, without translating those into Spinnaker kinds.  So whereas I'd think of Spinnaker "resources" as a server group, load balancer, etc, this endpoint is returning raw un-mapped Kubernetes resources.